File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
class Solution {
    public long[] solution(long x, int n) {
        long[] answer = new long[n];
        long temp = x;
        answer[0] = x;
        for (int i = 1; i < n; i++) {
            x += temp;
            answer[i] = x;
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.cs
public class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        int i = 0;
        int num = x;
        long xx = x;
        while (i < n) {
            answer[i] = xx;
            xx += num;
            i++;
        }
        return answer;
    }
}